src/index.js

```diff
@@ -0,0 +1,58 @@
+import Books from './Modules/books';
+import { addLocalStorage, getLocalStorage } from './Modules/localStorage';
+import displayBooks from './Modules/dispalybook';
+import './index.css';
+import date from './Modules/date';
+
+const form = document.querySelector('#form');
```

Comment:
all your code is perfect 😍 but I suggest if some code you aren't using in your project just cut them. So your code will look more clean🥰.